docs/docs/integrations/libraries/dbt/quickstart.md

```diff
@@ -12,86 +12,191 @@ Dagster orchestrates dbt alongside other technologies, so you can schedule dbt w
-  title="definitions.py"
+  title="src/dagster_dbt_quickstart/defs/transform/defs.yaml"
 />
 
 ## Adding upstream dependencies
 
-Often, you'll want Dagster to generate data that will be used by downstream dbt models. To do this, add an upstream asset that the dbt project will use as a source:
+Often, you'll want Dagster to generate data that will be used by downstream dbt models. To do this, scaffold an `assets.py` where custom assets can be defined:
+
+```bash
+dg scaffold defs dagster.asset assets.py
+```
+
+Then add an upstream asset:
 
 <CodeExample
-  path="docs_snippets/docs_snippets/guides/etl/transform-dbt/dbt_definitions_with_upstream.py"
+  path="docs_snippets/docs_snippets/guides/etl/transform-dbt/assets.py"
   language="python"
-  title="definitions.py"
+  title="src/dagster_dbt_quickstart/defs/assets.py"
+  startAfter="start_upstream_asset"
+  endBefore="end_upstream_asset"
 />
 
 Next, add a dbt model that will source the `raw_customers` asset and define the dependency for Dagster. Create the dbt model:
 
 <CodeExample
   path="docs_snippets/docs_snippets/guides/etl/transform-dbt/basic-dbt-project/models/example/customers.sql"
   language="sql"
-  title="customers.sql"
+  title="dagster_dbt_quickstart/src/basic-dbt-project/models/example/customers.sql"
 />
 
 Next, create a `_source.yml` file that points dbt to the upstream `raw_customers` asset:
 
 <CodeExample
   path="docs_snippets/docs_snippets/guides/etl/transform-dbt/basic-dbt-project/models/example/_source.yml"
   language="yaml"
-  title="_source.yml_"
+  title="dagster_dbt_quickstart/src/basic-dbt-project/models/example/source.yml_"
 />
 
-![Screenshot of dbt lineage](/images/integrations/dbt/dbt-lineage.png)
+![Screenshot of dbt lineage](/images/integrations/dbt/dbt-lineage-1.png)
 
 ## Adding downstream dependencies
 
-You may also have assets that depend on the output of dbt models. Next, create an asset that depends on the result of the new `customers` model. This asset will create a histogram of the first names of the customers:
+You may also have assets that depend on the output of dbt models. Within the `assets.py` file, create an asset that depends on the result of the new `customers` model. This asset will create a histogram of the first names of the customers:
 
 <CodeExample
-  path="docs_snippets/docs_snippets/guides/etl/transform-dbt/dbt_definitions_with_downstream.py"
+  path="docs_snippets/docs_snippets/guides/etl/transform-dbt/assets.py"
   language="python"
-  title="definitions.py"
+  title="src/dagster_dbt_quickstart/defs/assets.py"
+  startAfter="start_downstream_asset"
+  endBefore="end_downstream_asset"
 />
 
+![Screenshot of dbt lineage](/images/integrations/dbt/dbt-lineage-2.png)
+
 ## Scheduling dbt models
 
-You can schedule your dbt models by using the `dagster-dbt`'s `build_schedule_from_dbt_selection` function:
+You can schedule you dbt models using Dagster [schedules](/guides/automate/schedules). First scaffold a schedule:
```

Comment:
The schedules section is the only place where this diverges since components has no equivalent support for `build_schedule_from_dbt_selection`